lib/html-utils.js

```diff
@@ -41,9 +41,9 @@ const buildScriptElement = (obj) => {
         args.push('defer');
     }
 
-    if (notEmpty(obj.data)) {
-        Object.entries(obj.data).forEach(([key, value]) => {
-            args.push(`data-${key}="${value}"`);
+    if (Array.isArray(obj.data) && (obj.data !== 0)) {
```

Comment:
Is this supposed to be `obj.data.length !== 0`?

Reply:
Your right.